packages/backend/src/repositories/order.ts

```diff
@@ -56,30 +57,30 @@ export class OrderRepository {
       where: { id: productId },
     });
 
-    if (product.status === ProductStatus.ACTIVE) {
-      return this._dbClient.order.create({
-        data: {
-          productId,
-          buyerId,
-          cost: product.price,
-          status: OrderStatus.CREATED,
-        },
-        include: {
-          product: true,
-          buyer: {
-            select: {
-              id: true,
-              email: true,
-              firstName: true,
-              lastName: true,
-              phone: true,
-            },
-          },
-        },
-      });
+    if (product.status !== ProductStatus.ACTIVE) {
```

Comment:
We don't need to have business logic in repository, move it to the service please.